projects/components/src/table/cells/table-cell-injection.ts

```diff
@@ -36,6 +37,10 @@ export const createTableCellInjector = (
         provide: TABLE_CELL_DATA,
         useValue: cellData
       },
+      {
+        provide: TABLE_CELL_OPTIONS,
```

Comment:
Why do we need to inject it? Column config is already injected, can we pull from there?

Reply:
Yeah. I think will make the change then.